File: springboot/src/main/java/com/academia/entity/Pessoa.java
package com.academia.entity;

import java.sql.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.validation.constraints.NotEmpty;

import org.hibernate.validator.constraints.Length;

@Entity
public class Pessoa
{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_pessoa")
    private long id;

    @Column(nullable = false)
    @NotEmpty(message="O nome deverá ser informado!")
    @Length(min=5, max=200, message="O nome deverá ter de 5 a 200 caracteres")    
    private String nome;
    
    @Column
    private String cpf;
    
    @Column
    private String sexo;
    
    @Column
    private Date nascimento;
    
    @Column
    private Integer peso;

	@Column
    private Integer altura;
    
    @Column
    private String telefone;
    
    @Column
    private String observacao;
    
    @Column
    private String email;
    
    @Column
    private String funcao;
    
    @Column
    private Double salario;
    
    @Column
    private String turno;
    
    @Column
    private Integer tipocadastro;
    
    public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getCpf() {
		return cpf;
	}

	public void setCpf(String cpf) {
		this.cpf = cpf;
	}

	public String getSexo() {
		return sexo;
	}

	public void setSexo(String sexo) {
		this.sexo = sexo;
	}

	public Date getNascimento() {
		return nascimento;
	}

	public void setNascimento(Date nascimento) {
		this.nascimento = nascimento;
	}

	public Integer getPeso() {
		return peso;
	}

	public void setPeso(Integer peso) {
		this.peso = peso;
	}

	public Integer getAltura() {
		return altura;
	}

	public void setAltura(Integer altura) {
		this.altura = altura;
	}

	public String getTelefone() {
		return telefone;
	}

	public void setTelefone(String telefone) {
		this.telefone = telefone;
	}

	public String getObservacao() {
		return observacao;
	}

	public void setObservacao(String observacao) {
		this.observacao = observacao;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getFuncao() {
		return funcao;
	}

	public void setFuncao(String funcao) {
		this.funcao = funcao;
	}

	public Double getSalario() {
		return salario;
	}

	public void setSalario(Double salario) {
		this.salario = salario;
	}

	public String getTurno() {
		return turno;
	}

	public void setTurno(String turno) {
		this.turno = turno;
	}

	public Integer getTipocadastro() {
		return tipocadastro;
	}

	public void setTipocadastro(Integer tipocadastro) {
		this.tipocadastro = tipocadastro;
	}
   
}

File: flutter/lib/modelo/Pessoa.dart
class Pessoa {
  final String nome;
  final String cpf;
  final String sexo;
  final String nascimento;
  final int peso;
  final int altura;
  final String telefone;
  final String observacao;
  final String email;
  final String funcao;
  final double salario;
  final String turno;
  final int tipocadastro;

  Pessoa(
      this.nome,
      this.cpf,
      this.sexo,
      this.nascimento,
      this.peso,
      this.altura,
      this.telefone,
      this.observacao,
      this.email,
      this.funcao,
      this.salario,
      this.turno,
      this.tipocadastro);

  @override
  String toString() {
    // TODO: implement toString
    return 'Pessoa{$nome, $cpf, $sexo, $nascimento, $peso, $altura, $telefone, $observacao, $email, $funcao, $salario, $turno, $tipocadastro}';
  }

  Pessoa.fromJson(Map<String, dynamic> json)
      : nome = json['nome'],
        cpf = json['cpf'],
        sexo = json['sexo'],
        nascimento = json['nascimento'],
        peso = json['peso'],
        altura = json['altura'],
        telefone = json['telefone'],
        observacao = json['observacao'],
        email = json['email'],
        funcao = json['funcao'],
        salario = json['salario'],
        turno = json['turno'],
        tipocadastro = json['tipocadastro'];

  Map<String, dynamic> toJson() => {
        "nome": nome,
        "cpf": cpf,
        "sexo": sexo,
        "nascimento": nascimento,
        "peso": peso,
        "altura": altura,
        "telefone": telefone,
        "observacao": observacao,
        "email": email,
        "funcao": funcao,
        "salario": salario,
        "turno": turno,
        "tipocadastro": tipocadastro
      };
}
